### ISSUE TITLE
Missing requirements

### ISSUE THREAD

**Issue by Cyruz143** [mathoudebine/turing-smart-screen-python#21]
OS: Windows 10 x64
Branch: Release 2.0 alpha 3
GPU: Nvidia

`pip install -r requirements.txt`
`ModuleNotFoundError: No module named 'Cython'`
Assume this needs adding to the requirements.txt?

After manually installing Cython
```
PS C:\Apps\smartScreen> pip install -r requirements.txt
Requirement already satisfied: Pillow~=9.2.0 in c:\users\xxx\appdata\local\programs\python\python310\lib\site-packages (from -r requirements.txt (line 1)) (9.2.0)
Requirement already satisfied: pyserial~=3.5 in c:\users\xxx\appdata\local\programs\python\python310\lib\site-packages (from -r requirements.txt (line 2)) (3.5)
Requirement already satisfied: PyYAML~=6.0 in c:\users\xxx\appdata\local\programs\python\python310\lib\site-packages (from -r requirements.txt (line 3)) (6.0)
Requirement already satisfied: psutil~=5.9.1 in c:\users\xxx\appdata\local\programs\python\python310\lib\site-packages (from -r requirements.txt (line 4)) (5.9.1)
Requirement already satisfied: GPUtil~=1.4.0 in c:\users\xxx\appdata\local\programs\python\python310\lib\site-packages (from -r requirements.txt (line 5)) (1.4.0)
Collecting pyamdgpuinfo~=2.1.3
  Using cached pyamdgpuinfo-2.1.3.tar.gz (21 kB)
  Preparing metadata (setup.py) ... done
Using legacy 'setup.py install' for pyamdgpuinfo, since package 'wheel' is not installed.
Installing collected packages: pyamdgpuinfo
  Running setup.py install for pyamdgpuinfo ... error
  error: subprocess-exited-with-error

  × Running setup.py install for pyamdgpuinfo did not run successfully.
  │ exit code: 1
  ╰─> [12 lines of output]
      running install
      C:\Users\xxx\AppData\Local\Programs\Python\Python310\lib\site-packages\setuptools\command\install.py:34: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
        warnings.warn(
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-310
      creating build\lib.win-amd64-cpython-310\pyamdgpuinfo
      copying pyamdgpuinfo\__init__.py -> build\lib.win-amd64-cpython-310\pyamdgpuinfo
      running build_ext
      building 'pyamdgpuinfo._pyamdgpuinfo' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
error: legacy-install-failure

× Encountered error while trying to install package.
╰─> pyamdgpuinfo
```

**Comment by mathoudebine** [comment]
Hello! Thanks for raising this issue, i will have to add Cython and mark pyamdgpuinfo for Linux only (since the package doc says it's  only compatible with Linux)

**Comment by mathoudebine** [comment]
@Cyruz143 can you try the latest commit from branch `feature/system-monitor` ? the issue should be fixed there
Please note on Windows CPU temperature is not yet available, nor GPU data if you have an AMD GPU

**Comment by Cyruz143** [comment]
Checked latest commit, installing all requirements works and it runs the script without issues! Thanks for the quick fix.

Side note: You might want to make an issue template so when people report stuff they can specify environment/gpu etc.

**Comment by mathoudebine** [comment]
Thanks! Added the issue templates
Issue fixed in latest commit of feature/system-monitor, will be included in 2.0.0-alpha.5
